crates/kgraph_utils/benches/evaluation.rs

```diff
@@ -54,25 +55,25 @@ fn build_test_data(
 
     let profile_id = common_utils::generate_profile_id_of_default_length();
 
-    #[cfg(feature = "v2")]
-    let stripe_account = MerchantConnectorResponse {
-        connector_type: api_enums::ConnectorType::FizOperations,
-        connector_name: "stripe".to_string(),
-        id: common_utils::generate_merchant_connector_account_id_of_default_length(),
-        connector_account_details: masking::Secret::new(serde_json::json!({})),
-        disabled: None,
-        metadata: None,
-        payment_methods_enabled: Some(pms_enabled),
-        connector_label: Some("something".to_string()),
-        frm_configs: None,
-        connector_webhook_details: None,
-        profile_id,
-        applepay_verified_domains: None,
-        pm_auth_config: None,
-        status: api_enums::ConnectorStatus::Inactive,
-        additional_merchant_data: None,
-        connector_wallets_details: None,
-    };
+    // #[cfg(feature = "v2")]
```

Comment:
we can remove this code since the function flagged as cfg-feature `v1`

Reply:
we will uncomment this code once the tests are migrated, have kept it to retain the information